core/typedef.go

```diff
@@ -77,15 +77,6 @@ func (fn *Function) FieldSpec() (dagql.FieldSpec, error) {
 		Name:        fn.Name,
 		Description: formatGqlDescription(fn.Description),
 		Type:        fn.ReturnType.ToTyped(),
-
-		// NB: functions actually _are_ cached per-session, which matches the
```

Comment:
:eyes: can we keep this comment (in a new place) - just thinking about accidental removal, etc (though tests *should* catch it).